libraries/AP_L1_Control/AP_L1_Control.h

```diff
@@ -54,6 +54,7 @@ class AP_L1_Control : public AP_Navigation {
     void update_loiter(const class Location &center_WP, float radius, int8_t loiter_direction) override;
     void update_heading_hold(int32_t navigation_heading_cd) override;
     void update_level_flight(void) override;
+    void update_path(const class Location &position_on_path, Vector2f unit_path_tangent, float path_curvature, int8_t direction) override;
```

Comment:
we normally use "position" to mean an offset from the EKF origin so you could rename the arg to "location_on_path".

Reply:
the unit_path_tangent could be a const reference.

Reply:
Changed in b2cf0602e7 to c71535003a